src/main/java/com/sap/sgs/phosphor/fosstars/data/github/VendorDataMatcher.java

```diff
@@ -0,0 +1,131 @@
+package com.sap.sgs.phosphor.fosstars.data.github;
+
+import com.sap.sgs.phosphor.fosstars.nvd.Matcher;
+import com.sap.sgs.phosphor.fosstars.nvd.data.Affects;
+import com.sap.sgs.phosphor.fosstars.nvd.data.CVE;
+import com.sap.sgs.phosphor.fosstars.nvd.data.CVEDataMeta;
+import com.sap.sgs.phosphor.fosstars.nvd.data.NvdEntry;
+import com.sap.sgs.phosphor.fosstars.nvd.data.ProductData;
+import com.sap.sgs.phosphor.fosstars.nvd.data.Vendor;
+import com.sap.sgs.phosphor.fosstars.nvd.data.VendorData;
+import com.sap.sgs.phosphor.fosstars.tool.github.GitHubProject;
+import java.util.Objects;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+/**
+ * This is a matcher that check if vendor data from an NVD entry matched to
+ * a project's owner and name.
+ */
+public class VendorDataMatcher implements Matcher {
+
+  /**
+   * A logger.
+   */
+  private static final Logger LOGGER = LogManager.getLogger(VendorDataMatcher.class);
+
+  /**
+   * Creates a new matcher for a project.
+   *
+   * @param project The project.
+   * @return The new matcher.
+   */
+  public static VendorDataMatcher with(GitHubProject project) {
+    return new VendorDataMatcher(project);
+  }
+
+  /**
+   * A project to be checked.
+   */
+  private final GitHubProject project;
```

Comment:
Please move this line above the constructor.

Reply:
Okay.